docs/dev/lsp-extensions.md

```diff
@@ -579,3 +579,16 @@ This request is sent from client to server to open the current project's Cargo.t
 ```
 
 `experimental/openCargoToml` returns a single `Link` to the start of the `[package]` keyword.
+
+## Lookup workspace symbol search scope and kind
+
+**Issue:** https://github.com/rust-analyzer/rust-analyzer/pull/7698
+
+This request is sent from client to server to search for symbols constrained by a search
+scope and a symbol kind.
+
+**Method:** `workspace/symbol`
+
+**Request:** `WorkspaceSymbolParams`
+
+**Response:** `SymbolInformation[] | null`
```

Comment:
Let's actually specify what we change from the stock request. Take a look at how snippet text edit is specced: 

https://github.com/matklad/rust-analyzer/blob/459b0332d3a4905fa9bc93c484538e74d191c80b/docs/dev/lsp-extensions.md#L38-L62

Reply:
Done. Hopefully the added info is enough. I also couldn't figure out whether `export` is necessary for the interfaces.